pkgpanda/docker.windows/dcos-builder/setup-python27.ps1

```diff
@@ -1,15 +1,9 @@
 $PROGRAM = "Python-2.7.13"
-$DOWNLOAD_URL = "https://www.python.org/ftp/python/2.7.13/python-2.7.13.msi"
 $INSTALL_DIR = Join-Path $env:SystemDrive "Python27"
 $BIN_DIR =  Join-Path $INSTALL_DIR "bin"
 $INSTALL_ARGS = @("/qn", "/i")
 
-Write-Output "Downloading $PROGRAM from $DOWNLOAD_URL"
-$fileName = $DOWNLOAD_URL.Split('/')[-1]
-$downloadFilename = Join-Path $env:TEMP $fileName
-Invoke-WebRequest -UseBasicParsing -Uri $DOWNLOAD_URL -OutFile $downloadFilename
-
-$INSTALL_ARGS += $downloadFilename
+$INSTALL_ARGS += "c:\temp\python-2.7.13.msi"

```

Comment:
Using Python 2.7 will force pip to download Python2 versions of packages. We need Python 3.6

Reply:
using python 3.6 can be done of course, started to use what is in the repository and that was 2.7